apollo-router/src/cache/redis.rs

```diff
@@ -136,6 +136,14 @@ impl RedisCacheStorage {
         let url = Self::preprocess_urls(config.urls)?;
         let mut client_config = RedisConfig::from_url(url.as_str())?;
 
+        if let Some(username) = config.username {
+            client_config.username = Some(username);
```

Comment:
maybe check if the URL does not have already a username or password? Who takes precedence in that case?

Reply:
It's documented

Reply:
Fixed